packages/fxa-settings/.storybook/fxaTheme.js

```diff
@@ -3,7 +3,7 @@ import { create } from '@storybook/theming/create';
 export default create({
   base: 'light',
 
-  brandTitle: 'Firefox Accounts Storybook',
+  brandTitle: 'Mozilla Accounts Storybook',
   brandUrl: 'https://github.com/mozilla/fxa',
   brandImage: 'assets/logo.png',
```

Comment:
Nit: We might want to update the logo here too, even though it's just storybook.

Reply:
Logos updated. We might have to do a party branch here. So I'll push individual commits. We can squash them once this is R+.

Reply:
@dschom thanks for updating these logos in https://github.com/mozilla/fxa/pull/15923!

Reply:
@vpomerleau The fav icon changes have landed on main and I have rebased this branch on main.